### PR TITLE
feat(ProjectsUI):Changed Expand All To Expand Next Level and added alert message

### PR DESCRIPTION
**Pull request by afsahsyeda**
Signed-off-by: afsahsyeda <afsah.syeda@siemens-healhtineers.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

### Description:

- Expand All changed to 'Expand Next Level' to convey a clearer message as to how the nodes are loaded.
- An alert message saying 'All the levels are expanded!' is displayed after all the parent and child nodes are expanded successfully.

Issue: #1783 

### Suggest Reviewer
@akapti 

### How To Test?

1. Go to Projects portlet
2. Click on License Clearing tab
3. Click On 'Expand Next Level' button.

Screenshots:
image

### CODE REVIEW

**Comment by rudra-superrr** [comment]
Hi @afsahsyeda ,
When you click Collapse all button in order to collapse the expanded result, it works fine. But if you again click on the Expand next level button, it will expand the result to the max level and it does not show the message(All the levels are expanded) i.e. it will not expand one level at a time.

image

**Comment by rudra-superrr** [comment]
Testing was successful.

**Review by smrutis1 — APPROVED** [review]
Code looks good.

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/js/modules/ajax-treetable.js [hunk 1]
```diff
@@ -17,23 +17,48 @@ define('modules/ajax-treetable', [ 'jquery', 'utils/cssloader', /* jquery-plugin
...
                             data: data
                         }).done(function (result) {
                             node.row.data("children-loaded", true);
+                            
+                            if(/[a-z]/i.test(result)){
+                               resultCount++;
+                            }
+                            nodeCount++;
+
+                            if(nodeCount == Object.keys(node.tree).length){
```
Comment (akapti) [inline frontend/sw360-portlet/src/main/resources/META-INF/resources/js/modules/ajax-treetable.js:48]: please use correct indentation / formatting.
`if (condition) {` -> at line `52`, `43` and `29`.

**Review by akapti — CHANGES_REQUESTED** [review]
kindly incorporate the review comments.

one possible issue I found while testing:
- Collapsing node of `AttachmentUsage` tree-table might hide the warning message in `License Clearing` tree view table.

**Review by akapti — APPROVED** [review]
LGTM.

**Pull request merged**